src/utils/searchEvents.ts

```diff
@@ -0,0 +1,18 @@
+import { getDatabase, ref, get } from "firebase/database";
```

Comment:
@arunimaChintu What is the need of adding this file? This function is not even being used anywhere.